percona-server-8.3.advisories.yaml

```diff
@@ -21,6 +21,11 @@ advisories:
             componentType: apk
             componentLocation: /.PKGINFO
             scanner: grype
+      - timestamp: 2024-06-19T14:19:04Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
```

Comment:
We have a different type to indicate a version mismatch

Reply:
updated, let me know if that is the correct one